caasp-environment.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
This just needs to be replace with 3 --- to indicate the start which is optional of course so I don't mind either way.